portfolio/src/main/webapp/index.html

```diff
@@ -9,10 +9,40 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <p>My name is Kevin Shao, and this is my portfolio! I live in Los Angeles, California, 
+           but I went to a boarding school in New Jersey for high school and then ended up 
+           at MIT, where I just finished my first year.</p>
+
+        <p>In high school, I had the chance to try several sports, two of which have played 
+           a pretty big part in who I am: swimming and rowing.</p>
+
+        <p>In terms of programming, I love building everything, but I especially love games. 
+           One of my first big projects was a game of Chess, and some of my later projects have 
+           followed in a similar vein. I'm also really curious about Machine Learning, despite
+           not having any formal instruction in it, and I'm always ready to learn! For some of
+           my prouder projects, check out my <a href='http://github.com/K-Shao'>Github</a>! </p>
+
+        <p>In my free time, I really enjoy playing games of all sorts. I like the mental competition, 
+           as things rarely get boring when you're trying to outsmart someone who's trying to outsmart
+           you. My favorites include: 
+          <ul>
+            <li>Chess
+              <ul>
+                <li>There are a ton of variants of chess, too!</li>
+                <li><a href=https://en.wikipedia.org/wiki/Kriegspiel_(chess)>Kreigspiel</a>: imperfect information chess</li>
+                <li>Three check: you win once you give check three times</li>
+                <li>Atomic: captured pieces "explode"</li>
+                <li>...and <a href=https://en.wikipedia.org/wiki/List_of_chess_variants>many more</a></li>
+              </ul>
+            </li>
+            <li>Poker</li>
+            <li>Social deception games</li>
+            <li>Monopoly (for some reason)</li>
+          </ul>
+        </p>
+        <p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>
```

Comment:
I think this might not be aligned with the heading tag above? Since a lot of lines were deleted, you might want to shift everything two spaces to the left.